cmd/headscale/cli/acls.go

```diff
@@ -0,0 +1,60 @@
+package cli
+
+import (
+	"fmt"
+	
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(aclsCmd)
+	aclsCmd.AddCommand(listAclsCmd)
+
+}
+
+var aclsCmd = &cobra.Command{
+	Use:     "acls",
+	Short:   "Manage Access Control Lists (ACLs)",
+	Aliases: []string{"access-lists","acl"},
+}
+
+var listAclsCmd = &cobra.Command{
+	Use:     "list",
+	Short:   "List ACLs",
+	Aliases: []string{"ls","show"},
+	Run:      func(cmd *cobra.Command, args []string) {
+		output, _ := cmd.Flags().GetString("output")
+		if output == `` {
+			output = `json`
+		}
+		h, err := getHeadscaleApp()
+		if err != nil {
+			ErrorOutput(
+				err,
+				fmt.Sprintf("Error getting headscale app: %s", err),
+				output,
+			)
+
+			return
+		}
+		policy := h.GetACLPolicy()
```

Comment:
This won't do, you need to go through the protobuf API for this. Add message type and response in proto folder, generate code via `make generate` then edit `grpcv1.go` to add server-side part